app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
The feature based directories make sense, but that is a rather different case in my opinion. Having all the code in a single folder would clutter fast. It's still a style of organizing code, just a different approach to having for example controllers in their own folder. Both have their merits, but this really isn't about extracting directives to their own folder at all.

The background directive being in it's own module means that you could simply copy it to a different project if needed. The shared methods don't need to be restricted to a single app.

What negatives does the module have in your opinion?